pkgs/servers/gerbera/default.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv, fetchgit
+, cmake, pkg-config
+# required
+, libupnp, libuuid, pugixml, libiconv, sqlite, zlib, spdlog, fmt
+, pkgs
+# options
+, enableDuktape ? true
+, enableCurl ? true
+, enableTaglib ? true
+, enableLibmagic ? true
+, enableLibmatroska ? true
+, enableAvcodec ? false
+, enableLibexif ? true
+, enableExiv2 ? false
+, enableFFmpegThumbnailer ? false
+, enableInotifyTools ? true
+}:
+
+with stdenv.lib;
+let
+  optionOnOff = option: if option then "on" else "off";
+in stdenv.mkDerivation rec {
+  pname = "gerbera";
+  version = "1.5.0";
+
+  src = fetchgit {
```

Comment:
`fetchFromGitHub` is generally preferred for GitHub repositories. I think its more efficient or something (since it fetches a shallow tarball instead of the git repository). I don't recall the exact reasoning though.